Python/pywarpx/callbacks.py

```diff
@@ -159,19 +161,19 @@ def callbackfunclist(self):
             if not callable(result):
                 print("\n\nWarning: a call back was found that is not callable.")
                 if self.name is not None:
-                    print("For %s"%self.name)
+                    print("For %s" % self.name)
                 print("Only callable objects can be installed.")
                 print("It is possible that the callable's name has been overwritten")
                 print("by something not callable. This can happen during restart")
                 print("if a function name had later been used as a variable name.")
                 print(self.name)
-                if isinstance(f,str):
-                    print("The name of the call back is %s"%f)
+                if isinstance(f, str):
+                    print("The name of the call back is %s" % f)
```

Comment:
I guess another consistency might be to use `f-strings` everywhere that substitution is made. Could that be added to the Ruff checks?

Reply:
I am not sure if we can enforce it, but we can use it manually here :)
```suggestion
                    print(f"The name of the call back is {f}")
```

Reply:
If it can't be enforced, we also don't have to change this. There are several places where old replacement strings are still used.

Reply:
Let's start somewhere :D